src/hosted.js

```diff
@@ -95,7 +95,11 @@
     function load(Bramble) {
         Bramble.load("#bramble",{
             url: "index.html",
-            useLocationSearch: true
+            useLocationSearch: true,
+            enableCollaboration: true,
```

Comment:
enablecollaboration is used here: https://github.com/mozilla/brackets/pull/828/files#diff-878090be7f48acda4fa38b7440388092R288 
to make sure if someone wants to run brackets without collaboration, they can do that.

In this case, Collaboration.connect is not called.